src/utils/SpokeUtils.ts

```diff
@@ -72,11 +71,6 @@ export function isUnsafeDepositId(depositId: BigNumber): boolean {
   return maxSafeDepositId.lt(depositId);
 }
 
-// Determines if the input address (either a bytes32 or bytes20) is the zero address.
-export function isZeroAddress(address: Address): boolean {
-  return address.toBytes32() === ZERO_BYTES;
-}
```

Comment:
This function is redundant because it accepts `Address`, which already has an `isZeroAddress()` method.